teardown.py

```diff
@@ -14,9 +14,12 @@ class BaseRuntimeTeardownDocker(Test):
 
     def setUp(self):
 
+        script_dir = os.path.abspath(os.path.dirname(__file__))
+        self.log.info("running script from directory: %s" % script_dir)
+
         mockcfg = self.params.get('mockcfg')
         if mockcfg is None:
-            self.error("mock configuration file for building docker image must be supplied")
+            mockcfg = os.path.join(script_dir, "resources", "base-runtime-mock.cfg")
 
         if not mockcfg.endswith(".cfg"):
             self.error("mock configuration file %s must have the extension '.cfg'" %
```

Comment:
You're getting the mock config in a couple places. Please turn it into a function.